server/audit_math/shim.py

```diff
@@ -0,0 +1,130 @@
+"""shim.py: Shim code to interface between the calling conventions expected by the current
+bravo_sample_sizes() code with the API currently provided by the athena module.
+
+TODO: move minerva_sample_sizes into minerva.py and refactor to use exact vote counts,
+like we did with compute_risk.
+Then this file can be eliminated.
+"""
+
+import logging
+import math
+from typing import Any
+from decimal import Decimal
+from athena.audit import Audit  # type: ignore
+
+
+def make_election(risk_limit, p_w: float, p_r: float) -> Any:
+    """
+    Transform fractional shares to an athena Election object.
+
+    Inputs:
+        risk_limit      - the risk-limit for this audit
+        p_w             - the fraction of vote share for the winner
+        p_r             - the fraction of vote share for the loser / runner-up
+    """
+
+    # calculate the undiluted "two-way" share of votes for the winner
+    p_wr = p_w + p_r
+    p_w2 = p_w / p_wr
+
+    contest_ballots = 100000
```

Comment:
I assume this code will go away once it gets integrated into minerva.py? Since we have the vote counts there, I don't see why you would need to keep fudging it like this

Reply:
Yes - that will go away, but since we aren't actually using the code at this point except for the first round with no samples, I may just leave this for a future PR, if that's ok.

Reply:
Done